examples/duet/vae/AutoEncoder_Syft_Data_Scientist.ipynb

```diff
@@ -1,8 +1,395 @@
 {
```

Comment:
<p>Same here add the cell above with the "data scientist" part.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenMined/PySyft/pull/5299/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
Done